GiftGivingGenerator.API/Controllers/DrawingResultsController.cs

```diff
@@ -44,20 +46,19 @@ public ActionResult SendEmail([FromRoute] Guid eventId)
 		{
 			return BadRequest(ModelState);
 		}
-		
-		//TODO: fix - organizer is now person type
-		// var @event = _eventRepository.Get(eventId);
-		// var organizer = _organizerRepository.Get(@event.OrganizerId);
-		//
-		// var drawingResultIds = _repository.GetDrawingResultsByEventId(eventId).Select(x=>x.Id);
-		// var body = "http://localhost:5036/DrawingResults/" + string.Join("\nhttp://localhost:5036/DrawingResults/", drawingResultIds);
-		//
-		// _mail.Send($"{organizer.Email}", $"Links to drawing results '{@event.Name}'", $"{body}");
-		
+
+		var @event = _eventRepository.Get(eventId);
+		var organizer = _personRepository.Get(@event.OrganizerId);
+
+		var drawingResultIds = _repository.GetDrawingResultsByEventId(eventId).Select(x => x.Id);
+		var body = "http://localhost:5036/DrawingResults/" + string.Join("\nhttp://localhost:5036/DrawingResults/", drawingResultIds);
```

Comment:
- move the base host to app settings
- don't concatenate strings (use string interpolation instead)

Reply:
done